specification/compute/resource-manager/Microsoft.Compute/stable/2021-12-01/disk.json

```diff
@@ -2590,6 +2602,28 @@
         ]
       }
     },
+    "DataAccessAuthMode": {
+      "type": "string",
+      "description": "Add additional AAD authentication requirement when accessing export/upload URL.",
+      "enum": [
+        "AzureActiveDirectory",
+        "None"
+      ],
+      "x-ms-enum": {
+        "name": "DataAccessAuthMode",
+        "modelAsString": true,
+        "values": [
+          {
+            "value": "AzureActiveDirectory",
+            "description": "When export/upload URL is used additional authentication is performed, to check if the user has necessary RBAC access. Please refer to aka.ms/{placeholder}."
```

Comment:
When export/upload URL is used, the system checks if the user has an identity in Azure Active Directory and has necessary permissions to export/upload the data.

Reply:
Please refer to [aka.ms/DisksAzureADAuth](https://aka.ms/DisksAzureADAuth)

Reply:
updated